### DISCUSSION TITLE
ReadItLater Feature Request

### DISCUSSION THREAD

**Discussion by ashtom50** [DominikPieper/obsidian-ReadItLater#31] [Ideas]
Hi. Very new to GitHub, sorry if im dumping this in the wrong place.
Really loving this plugin.
It would be really good if the youtube template could also clip the youtube channel and video description.
This would allow me to have my template looking something like this,

[[ReadItLater]] [[Youtube]]
# [%videoTitle%]
## Channel: [%YoutubeChannel%]
(%videoURL%)
%videoPlayer%
## Notes:

So I could simply click read it later, fill in my notes on the video and its done.

**Comment by adamluckdev** [comment]
All requested template variables are available, [see the reference](https://github.com/DominikPieper/obsidian-ReadItLater?tab=readme-ov-file#youtube).
